### ISSUE TITLE
Error when compiling with --enable-libvmaf

### ISSUE THREAD

**Issue by Elikay** [rdp/ffmpeg-windows-build-helpers#368]
When compiling with --enable-libvmaf added to the script, the cross compiler gives the error `ERROR: libvmaf >= 1.3.9 not found using pkg-config`

When checking pkg-config manually, libvmaf is there, and is version 1.3.11. I've also tried compiling just the standard ffmpeg source code with `./configure --enable-libvmaf --enable-version3` and it worked completely fine. I'm only compiling for Windows 64-bit, but I'd assume this is an issue for 32-bit as well.

**Comment by rdp** [comment]
try checking your ffmpeg's ffbuild/fconfig.log file

**Comment by Elikay** [comment]
This is the only thing relevant to libvmaf.
`/home/elikay/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/x86_64-w64-mingw32-gcc -Wl,--nxcompat,--dynamicbase -Wl,--high-entropy-va -Wl,--as-needed -Wl,--pic-executable,-e,mainCRTStartup -Wl,--image-base,0x140000000 -I/home/elikay/ffmpeg-win$`
`require_pkg_config libvmaf libvmaf >= 1.3.9 libvmaf.h compute_vmaf`
`check_pkg_config libvmaf libvmaf >= 1.3.9 libvmaf.h compute_vmaf`
`test_pkg_config libvmaf libvmaf >= 1.3.9 libvmaf.h compute_vmaf`
`pkg-config --exists --print-errors libvmaf >= 1.3.9`
`Package libvmaf was not found in the pkg-config search path.`
`Perhaps you should add the directory containing 'libvmaf.pc'`
`to the PKG_CONFIG_PATH environment variable`
`No package 'libvmaf' found`
`ERROR: libvmaf >= 1.3.9 not found using pkg-config`
I believe it might be because vmaf isn't technically apart of ffmpeg, but is instead a separate library made by netflix. I have it installed locally on my machine, so it works fine when I build normally like I said before, it's only an issue when I use this script to build for Win64. Is it because the script doesn't build vmaf itself, and is looking somewhere else, despite using pkg-config? I don't fully understand how the script works, but it seems that it isolates itself from the rest of my system? Is there a way to force it to use my locally installed vmaf, or can I make the script compile vmaf itself?

**Comment by rdp** [comment]
Right it builds its own copies of all the libraries (local copies)...I
could look at it sometime if you'd like...

**Comment by Elikay** [comment]
Well is there anything I could write in the script so it'll compile it's own version of vmaf, or to use my installed version?

**Comment by rdp** [comment]
OK I finally co-erced it into building, try with the latest script, see if it works, untested, cheers!

**Comment by fand** [comment]
I saw same error message when I run `./cross_compile_ffmpeg.sh --build-ffmpeg-shared=y` on macOS. 
(btw this repo is great, thx rdp B)

**Comment by rdp** [comment]
It should compile and include libvmaf as of...like today LOL.  Make sure
you're on the latest, are you?

**Comment by fand** [comment]
yep I'm on 882e6ab, what I did is:
```sh
git clone https://github.com/rdp/ffmpeg-windows-build-helpers/
brew install wget cvs hg yasm autogen automake autoconf cmake libtool xz pkg-config nasm bzip2 autoconf-archive p7zip
./cross_compile_ffmpeg.sh --build-ffmpeg-shared=y
```

`pkg-config` works well on other repo, maybe some dependencies missing on my mac...?
I removed code for `libvmaf` from `./cross_compile_ffmpeg.sh` and now I'm running it again

**Comment by rdp** [comment]
nuke the sandbox after getting latest git master perhaps?  If not I'll try
it on os x sometime...

**Comment by rdp** [comment]
whoa vmaf was way different in OS X. Weird makefile.  Hopefully fixed in 29f2d9f4f8b thank you for the report.
